pkg/templates/templates_generated.go

```diff
@@ -6865,6 +6860,23 @@ function Postpone-RestartComputer
     Write-Log "Created an one-time task to restart the VM"
 }
 
+function Create-Directory
+{
+    Param(
+        [Parameter(Mandatory=$true)][string]
+        $Folder,

```

Comment:
not blocking.. Directory and Folder are two marks referring to same thing.

If the function name is already Create-Directory, the para could be 'Name' and 'Usage'

Reply:
@karataliu Rename to "FullPath" since it is a full path but not a name.